src/components/toast/toast.component.js

```diff
@@ -44,6 +50,8 @@ const Toast = React.forwardRef(
 
     const focusedElementBeforeOpening = useRef();
 
+    const [tabIndex, setTabIndex] = useState(!disableAutoFocus && 0);
+
```

Comment:
Thanks for making the change but I don't like this line I'm afraid. This will set the initial `tabIndex` to `false` when `disableAutoFocus` is true, which is meaningless in HTML. (And likely gets ignored by the browser, so this will still work, but it's not something we should do.)

I would do `const [tabIndex, setTabIndex] = useState(disableAutoFocus ? 0 : undefined)` but happy to hear alternatives - but I do think we should ensure the value is `undefined` when `disableAutoFocus` isn't set.

Reply:
Valid point, I'd misunderstood how non-integer values would be treated